jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -379,7 +379,10 @@ def _auto_slider_range(self, event=None):
         ndec = -np.log10(half_range)
         if ndec > 0:
             # round to at least 2 digits, or the first significant digit
-            ndec = np.max([2, int(np.ceil(ndec))])
+            try:
+                ndec = np.max([2, int(np.ceil(ndec))])
```

Comment:
I feel like @kecnry already fixed this in #1221 but in a different way.

Reply:
Good call, I'll revert this fix in favor of that one.